### PR TITLE
(SIMP-8956) simp-adapter EL8 support

### PR DESCRIPTION
**Pull request by lnemsick-simp**
- Added EL8 support
  - Workaround dnf uninstall nuances on EL8
    - Use 'Recommends' in lieu of 'Requires' so that the git
      package is not uninstalled if the simp-adapter package
      is uninstalled.
    - simp_rpm_helper now checks for the 'git' or 'rsync' executables
      and fails if either cannot be found. This is necessary because
      'Recommends' does not ensure 'git' and 'rsync' are available.

- Updated tests:
  - Removed EL6 nodes
  - Added EL8 nodes
  - Split out nodes into individual node sets, because the steps
    required to build the simp-adapter RPM (e.g., install rvm and build
    a ruby version) take a great deal of time and are likely to cause
    ssh timeouts in GitLab.
  - Restrict simp-adapter upgrade test to nodes with EL < 8, as the
    legacy SIMP 6 repo at download.simp-project.com only supports
    EL6 and EL7.

SIMP-8956 #close

### CODE REVIEW

**Comment by lnemsick-simp** [comment]
Investigating el8 FIPS mode slow RPM installs and repo failure in GitLab tests.

**Comment by lnemsick-simp** [comment]
Running the EL8 FIPS test locally took many hours to run (instead of minutes) and ultimately failed with the first installation of the first test RPM. The failure was due to the test RPMs not having signatures.  Entered a separate ticket for this problem: https://simp-project.atlassian.net/browse/SIMP-9016

File: spec/acceptance/suites/default/00_basic_test_spec.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require 'spec_helper_acceptance'
 
+TEST_RVM_VERSION = (ENV.key?('TEST_RVM_VERSION') ? ENV['TEST_RVM_VERSION'] : '2.5.8')
```
Comment (op-ct) [inline spec/acceptance/suites/default/00_basic_test_spec.rb:3]: _(Informational only; no need to change:)_ A simpler idiomatic equivalent:
```suggestion
TEST_RVM_VERSION  = ENV['TEST_RVM_VERSION'] || '2.5.8'
```

**Review by op-ct — APPROVED** [review]
👍 👁️

**Pull request merged**